### PR TITLE
Manage credentials

### PR DESCRIPTION
**Pull request by chaws**
@mwasilew I guess we can handle credentials this way

### CODE REVIEW

File: squad_client/manage.py [hunk 1]
```diff
@@ -20,6 +29,15 @@ def main():
         parser.print_help()
         return -1
 
+    squad_host = args.squad_host or os.getenv('SQUAD_HOST')
+    if squad_host:
+        try:
+            squad_token = args.squad_token or os.getenv('SQUAD_TOKEN')
```
Comment (mwasilew) [inline squad_client/manage.py:35]: I would check if squad_token isn't empty.

File: squad_client/manage.py [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
 
 
+logger = logging.getLogger()
+
+
 def main():
+
     parser = argparse.ArgumentParser(prog='./manage.py')
     parser.add_argument('--debug', action='store_true', help='display debug messages')
+    parser.add_argument('--squad-host', help='SQUAD host, example: https://qa-reports.linaro.org')
```
Comment (mwasilew) [inline squad_client/manage.py:21]: make it 'required' ?

Comment (chaws) [reply squad_client/manage.py:21]: Yeah, why not, fixing it

File: squad_client/manage.py (hunk 1 above)
Comment (chaws) [reply squad_client/manage.py]: just for the record, there's already a check for empty token here: https://github.com/Linaro/squad-client/blob/master/squad_client/core/api.py#L45

**Pull request merged**